src/unpacker/unpack.py

```diff
@@ -26,10 +26,10 @@ def unpack(self, current_fo: FileObject):
         Recursively extract all objects included in current_fo and add them to current_fo.files_included
         '''
 
-        logging.debug('[worker {}] Extracting {}: Depth: {}'.format(self.worker_id, current_fo.uid, current_fo.depth))
+        logging.debug(f'[worker {self.worker_id}] Extracting {current_fo.uid}: Depth: {current_fo.depth}')
 
         if current_fo.depth >= self.config.getint('unpack', 'max_depth'):
-            logging.warning('{} is not extracted since depth limit ({}) is reached'.format(current_fo.uid, self.config.get('unpack', 'max_depth')))
+            logging.warning(f"{current_fo.uid} is not extracted since depth limit ({self.config.get('unpack', 'max_depth')}) is reached")
```

Comment:
```suggestion
    max_depth = self.config.getint('unpack', 'max_depth')
    if current_fo.depth >= max_depth:
            logging.warning(f'{current_fo.uid} is not extracted since depth limit ({max_depth)}) is reached')
```